packages/export/src/utils.js

```diff
@@ -1,6 +1,13 @@
 import F from 'futil'
 import _ from 'lodash/fp.js'
 
+export const writeStreamData = async (stream, writeStreamData) => {
```

Comment:
I think this function belongs in /export/src/excel.js. Your test for it lives in /export/src/excel.test.js and it is not used anywhere outside of the excel.js file.